packages/flutter/lib/src/widgets/scroll_position.dart

```diff
@@ -466,6 +466,17 @@ class ScrollPosition extends ViewportOffset {
   ScrollActivity get activity => _activity;
```

Comment:
honestly i'd rather we moved towards making activity private, i don't really remember why it's public...